materials/demo.py

```diff
@@ -0,0 +1,39 @@
+"""Demonstration."""
+import numpy as np
+import matplotlib.pyplot as plt
+import material
+
+def main():
+    # pylint: disable=no-member
```

Comment:
He who discards the holy words

Reply:
pylint disable removed in https://github.com/mvernacc/material-properties-interchange/pull/1/commits/11a5f967c9f4202695d869ae85e6467e37e0a353